### ISSUE TITLE
How do i use a 3rd psmove controller for orientation in normal games or steamvr

### ISSUE THREAD

**Issue by Naming-is-hard** [psmoveservice/PSMoveService#656]
Hi,i have a Doogee x55 cellphone,and sadly it doesnt have a gyroscope,and i was wondering if i could use a 3rd psmove controller for orientation and if a can ,please tell me how,thanks.

**Comment by NikitaShokin** [comment]
Yes, you could use 3rd controller. You can use freepie (Read FREEPIE USERS on realise page)

**Comment by damiano-cmd** [comment]
what releases?
